Updated database and added new code

YoYoK3nny opened the pull request:
Updated the item database to that of CrossCode v1.0.0-10.
Also added a section to edit the list of favorite consumables. (My save was bugged for some reason and was auto-favoriting things that shouldn't be put there)

YoYoK3nny committed:
Updated database and added new code

Updated the item database to that of CrossCode v1.0.0-9.
Also added a section to edit the list of favorite consumables. (My save was bugged for some reason and was auto-favoriting things that shouldn't be put there)

YoYoK3nny committed:
Update database, added batch script.

Updated the item database to that of 1.0.0-10.
Also added a Windows Batch script to update the "itemj.js" if the "item-database.json" has been copied from the game assets into the editor's folder.

YoYoK3nny committed:
Limit Favorite Consumables

Limited the list of Favorite Consumables to 12. Now if the save happens to have more than that, the extras are simply discarded.

YoYoK3nny committed:
Fixed bug with Favorites

Fixed a bug where previous Favorites were still there after deleting some Favorites.

YoYoK3nny committed:
Resolved conflict in PR

Primajin commented:
Check out https://github.com/Primajin/CCSaveEdit has latest version and is easy to upgrade - also I'd like to pull in your fixes.

YoYoK3nny commented:
Feel free to use my fixes if you'd like @Primajin

YoYoK3nny committed:
Slight code cleanup.

Removed a leftover line of code from before the limiting of the favourites list.

Actually surprised it didn't break anything with regards to the list of Favourite Consumables...

Primajin commented:
Pulled in https://github.com/Primajin/CCSaveEdit/pull/3
